feg/protos/s8_proxy.proto

```diff
@@ -13,6 +13,8 @@ limitations under the License.
 
 syntax = "proto3";
 
+import "lte/protos/oai/spgw_state.proto";
```

Comment:
such inter module dependencies may be tricky to manage/maintain. Is there a way to  redefine TrafficFlowTemplate here or maybe put them in orc8r/protos?

Reply:
We can redefine just copying all the objects and nested object in this file.

But my concern is that then MME will have to get those feg.TFT objects and parse them out into the same lte.TFT objects. And this will work now, but the day we add a field in one and forget in the other one this will break.


Not sure if you will have an answer to this question, but after having tried different combinations my question is: why protos needed be defined per gateway instead of defining all of them in a separate independent module? Protos are interfaces between gateways so they don't really belong to a gateway. Or in other words, why protos were designed to be separated if they can share so many objects?

Reply:
well, some modules may have nothing to do with other module protos, but it obviously not the case with LTE, FEG & CWF